### DISCUSSION TITLE
Editable table UI

### DISCUSSION THREAD

**Discussion by redeemerr** [filamentphp/filament#5137] [Help] [answered]
Hi Guys, 

I am looking for a way to have a editable tables is there a plugin for that ? 

something like this: 
 https://codesandbox.io/s/material-ui-editable-tables-wsp0c

Best Regards

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/tables/columns/getting-started#available-columns

**Reply by redeemerr** [reply]
How did i miss that lol, Thx @danharrin
